src/blocks/main/footer/__menu/footer__menu.css

```diff
@@ -0,0 +1,17 @@
+.footer__menu {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  display: flex;
+}
+
+/* .footer__menu:last-of-type {
+  margin-right: 0;
+} */
```

Comment:
Although most modern minimizers auto-remove comments, it is better to remove temporary code before committing.